File: aat-lib/src/test/java/ch/bailu/aat_lib/TestTest.java
package ch.bailu.aat_lib;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.AfterEach;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

public class TestTest {

    boolean initValue = false;

    @BeforeEach
    public void init() {
        initValue = true;
    }

    @Test
    public void testTest() {
        assertEquals(true, initValue);
    }

    @AfterEach
    public void reset() {
        initValue = false;
    }
}

File: aat-lib/src/test/kotlin/ch/bailu/aat_lib/util/TestTest.kt
package ch.bailu.aat_lib.util

import org.junit.jupiter.api.AfterEach
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test

class TestTest {
    var initValue: Boolean = false

    @BeforeEach
    fun init() {
        initValue = true
    }

    @Test
    fun testTest() {
        Assertions.assertEquals(true, initValue)
    }

    @AfterEach
    fun reset() {
        initValue = false
    }
}
